packages/react/src/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
Is there any way to use `git mv` so that the changes here are easier to parse?

https://github.com/carbon-design-system/carbon/blob/main/docs/guides/adding-component-types.md#steps-to-provide-baseline-type-definitions-for-components

>Change extension to .tsx via git (do not simply rename)

>e.g. `git mv packages/react/src/components/ComponentName/ComponentName.js packages/react/src/components/ComponentName/ComponentName.tsx`

Reply:
I did use `git mv` but for some reason it reverted again. I'll try again.

Reply:
I've tried renaming it again, but it seems the diff is too big for git to recognize it as a rename.